plop-templates/Component/Component.tsx.hbs

```diff
@@ -0,0 +1,36 @@
+import React, { ReactNode } from "react";
+
+import classNames from "classnames/bind";
+import styles from "./{{pascalCase name}}.module.css";
+
+const cx = classNames.bind(styles);
+
+type Props = {
+  /**
+   * Child node(s) that can be nested inside component.
+   */
+  children: ReactNode;
+  /**
+   * How the text should be aligned.
+   */
+  align: "left" | "right";
+};
+
+/**
+ * This component is a replica of the template created by plop found in plop-templates.
+ * TODO: Find a way to keep these 2 files in sync.
+ */
+function {{pascalCase name}}({ children, align }: Props) {
+  return (
+    <div className={cx("{{camelCase name}}--container", `{{camelCase name}}--align-${align}`)}>
```

Comment:
This line is one of two locations where I was trying to use `paramCase`.

Reply:
were you thinking of kebab-case?